CustomUIButton.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomUIButton.swift
+//  Connnect
+//
+//  Created by Stella Su on 11/22/16.
+//  Copyright © 2016 iOS-Connect. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CustomUIButton: UIButton{
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.backgroundColor = .clear
+        self.layer.cornerRadius = 10
+        self.layer.borderWidth = 2
+        
+        self.layer.borderColor = self.tintColor.cgColor
+
+    }
+    
```

Comment:
Then to be really thorough we should also override the convenience initializer for UIButton, so users of this class always get the button they expect. 

I thing the initializer I'm thinking of is something like `init(buttonStyle: UIButtonStyle)`